### ISSUE TITLE
Not working for large files like above 1 GB

### ISSUE THREAD

**Issue by SwapnilNSDN** [pedroSG94/metadata-spherical-injector#2]
This library is not working for large files and it is very time taking too I please provide progress in percentage so I can show there.

**Comment by pedroSG94** [comment]
I assume that the problem is with video files. Right?
I will check it but the library is in C++ nad I don't know so much about it.

**Comment by SwapnilNSDN** [comment]
Yes I have video files upto 15GB and I want to inject 360Metadata init but this library work upto 1.8GB.
